website/docs/flow/stages/email/index.mdx

```diff
@@ -25,6 +25,10 @@ return True
 
 You can also use custom email templates, to use your own design or layout.
 
+:::info
+Starting with authentik 2024.1, it is possible to create `.txt` files with the same name as the `.html` template, which will be attached to the email.
```

Comment:
> which will be attached to the email

It's a bit unclear that HTML files are attached whereas txt files will be the body of the email.